src/kube-runtime/src/runtime.d/install_group.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+PAI_WORK_DIR=/usr/local/pai
+CACHE_ROOT_DIR=${PAI_WORK_DIR}/package_cache
+
+ubuntu_is_successfully_installed(){
+  for package in $1
+  do
+    dpkg -l $package &> /dev/null
+    if [ $? -ne 0 ]; then
+      return 1
+    fi
+  done
+  return 0
+}
+
+if [ $# -ne 1 ]; then
+  echo "Usage: bash -x install_group.sh <group_name>"
+  exit 1
+else
+  name=$1
+fi
+
+if cat /etc/issue | grep "Ubuntu 16.04" &> /dev/null ; then
+  os='ubuntu16.04'
+elif cat /etc/issue | grep "Ubuntu 18.04" &> /dev/null ; then
+  os='ubuntu18.04'
+else
+  echo "[package_cache] This os doesn't support package cache!"
+  exit 1
+fi
+if [ -d $CACHE_ROOT_DIR"/${name}-${os}" ]; then
```

Comment:
Do we need to check arch such as `x64` or `i386` here?

Reply:
A 64-bit system cannot use a 32-bit docker image, and vice versa. Since OpenPAI is based on 64-bit os, I think there is no need to check the system arch here.

Reply:
> Do we need to check arch such as `x64` or `i386` here?

A 64-bit system cannot use a 32-bit docker image, and vice versa. Since OpenPAI is based on 64-bit os, I think there is no need to check the system arch here.

Reply:
We may support 64bit only now, but 
- it is better to clarify it in the documents (e.g. Only `ubuntu16. 04` and `ubuntu18. 04` with 64bit arch are supported)
- have a check here and alert a warning if 32bit arch (rarely?) here

Reply:
it is a good idea to check the environment. 
maybe we can fallback to online apt-get when 32bit is required?

Reply:
Done. If a job runs on a 32-bit ubuntu system and the required packages are not installed, kube-runtime will use `apt-get` to install them.